src/status_im/contexts/profile/login/events.cljs

```diff
@@ -177,7 +177,7 @@
  :profile.login/get-user-password-success
  (fn [{:keys [db]} [password]]
    (when password
-     (let [{:keys [key-uid password]} (:profile/login db)]
+     (let [{:keys [key-uid]} (:profile/login db)]
```

Comment:
that looks strange, do you know why?

Reply:
Hi @flexsurfer, Thank you for reviewing PR.

Removed let block, is it looks ok now? (Or you meant something else)

Reply:
btw, interestingly we had somewhat same code before [refactor](https://github.com/status-im/status-mobile/pull/19005), but due to rf/merge password was updated before using

![main](https://github.com/status-im/status-mobile/assets/17097240/2fac6c48-efbe-4db8-8bed-821c0b6361e2)
![main2](https://github.com/status-im/status-mobile/assets/17097240/1dda1922-cefd-4535-9407-442d47a5e1fb)